### PR TITLE
chore: raw image support in cv2 format for dd_client

### PR DESCRIPTION
**Pull request by beniz**
Added `from_img` parameter to `post_predict` to pass raw images, goes along with `use_base64=True`

### CODE REVIEW

File: clients/python/dd_client/__init__.py [hunk 1]
```diff
@@ -255,6 +258,7 @@ def post_predict(
         parameters_output,
         use_base64=False,
         index_uris=[],
+        from_img=False
```
Comment (Bycob) [inline clients/python/dd_client/__init__.py:261]: you could remove "from_img" parameter and just detect whether the image contained in `data` is a `np.array` or a string

Comment (beniz) [reply clients/python/dd_client/__init__.py:261]: Could be. One drawback is that it requires importing numpy... Already I don´t like having to import cv2 into the client.

Comment (beniz) [reply clients/python/dd_client/__init__.py:261]: FTR, it's testing for `np.ndarray`.

Comment (beniz) [reply clients/python/dd_client/__init__.py:261]: cv2 imports numpy anyways...

Comment (Bycob) [reply clients/python/dd_client/__init__.py:261]: cv2 image is a numpy array

**Pull request closed without merging**